shared/utility/input/LimbID.cpp

```diff
@@ -25,20 +25,27 @@ namespace utility::input {
     using ServoID = utility::input::ServoID;
 
     LimbID::LimbID(std::string const& str) : value(Value::UNKNOWN) {
-        if (str == "UNKNOWN")
+        if (str == "UNKNOWN") {
             value = Value::UNKNOWN;
-        else if (str == "LEFT_LEG")
+        }
+        else if (str == "LEFT_LEG") {
             value = Value::LEFT_LEG;
-        else if (str == "RIGHT_LEG")
+        }
+        else if (str == "RIGHT_LEG") {
             value = Value::RIGHT_LEG;
-        else if (str == "LEFT_ARM")
+        }
+        else if (str == "LEFT_ARM") {
             value = Value::LEFT_ARM;
-        else if (str == "RIGHT_ARM")
+        }
+        else if (str == "RIGHT_ARM") {
             value = Value::RIGHT_ARM;
-        else if (str == "HEAD")
+        }
+        else if (str == "HEAD") {
             value = Value::HEAD;
-        else
+        }
+        else {
             throw std::runtime_error("String did not match any enum for LimbID");
+        }
```

Comment:
If you want for this you can turn off clang-format and make if statements inline. Makes it much easier to read